Civi/Payment/PropertyBag.php

```diff
@@ -777,9 +777,12 @@ public function setIsBackOffice($isBackOffice, $label = 'default') {
    *
    * @param string $label
    *
-   * @return bool|null
+   * @return bool
    */
   public function getIsRecur($label = 'default'):bool {
+    if (!$this->has('isRecur')) {
```

Comment:
@artfulrobot I think in this case  if isRecur is not set it's false

Reply:
I don't think this is the right place to add defaults. In most cases defaults don't make sense but for `isRecur` and probably `isBackOffice` I think defaults do make sense. But they should be set in `protected $props = ['default' => []];` so that they are consistently retrieved (eg. by `->getter()`).

Reply:
OK - I'll defer to what @artfulrobot thinks on that

Reply:
@artfulrobot thanks - I think we should merge this then - it seems like a soft point but I would say in the case of a true boolean field the absence of true is false - and I think this and isBackOffice are both expected to be true booleans